test/seriesController/seriesScenarios.ts

```diff
@@ -53,10 +53,8 @@ contract("Series Scenarios", (accounts) => {
     // Give Alice the collateral
     await collateralToken.mint(aliceAccount, mintAmount)
 
-    // Erase all previous rounds
-    await deployedMockPriceOracle.reset()
     const oraclePrice = 22_000 * 10 ** 8 // 22k
-    await deployedMockPriceOracle.addRound(oraclePrice, expiration, expiration)
+    await deployedMockPriceOracle.setLatestAnswer(oraclePrice)
```

Comment:
@MysticDakra have you done this change? I can not remember, if it was done by myself.

Reply:
I didnt make any changes I just pulled these in from your other branch.

Reply:
@MysticDakra, I realised what happened. 
The removed code was added to the main in commit 7b97bba095d8706fd0e67e2fa5f5167af02c2637.
Unfortunately, this branch was forked before that commit and does not contain the new changes after this commit. (It was not synced with main)
So it overrides the new changes with the old ones.